README.markdown

```diff
@@ -200,7 +200,15 @@ The `owncloud` class configures all possible options for this module. With defau
 
 ##### `datadirectory`
 
-Sets the directory user data will be stored in. It is not recommended to keep this in the default location (as a sub directory of the application document root) and it should be moved out of the document root before making your ownCloud instance accessible via the internet. Defaults to '/var/www/owncloud/data' on Debian based systems and `/var/www/html/owncloud/data` on RedHat based systems.
+Sets the directory user data will be stored in. It is not recommended to keep this in the default location (as a sub directory of the application document root) and it should be moved out of the document root before making your ownCloud instance accessible via the internet. Defaults to `/var/www/owncloud/data` on Debian based systems and `/var/www/html/owncloud/data` on RedHat based systems.
+
+##### `admin_user`
+
+Optionally set the admin user in the ownCloud configuration (using `admin_pass` as the password). If not set, OwnCloud will ask for admin credentials upon first connection.  
+
+##### `admin_pass`
+
+Optionally set the admin password in the ownCloud configuration. 
```

Comment:
Could you move these parameter descriptions above the `datadirectory` description to keep in alphabetical order please.